yaserde_derive/src/de/expand_enum.rs

```diff
@@ -24,6 +24,20 @@ pub fn parse(
 
   let flatten = root_attributes.flatten;
 
+  let get_name = if let Some(tag) = &root_attributes.tag {
```

Comment:
`get_name` is confusing name (it seems like a get/function pointer variable).

suggestion: `element_name` ?

Reply:
`element_name` does seems better, i switched to it.